browse-building-floors/src/main/res/layout/browse_floors_dropdown_item.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do we need this layout file?
Since in the MainActivity.kt we are using `android.R.layout.simple_list_item_1` layout to display the levels.